src/index.html

```diff
@@ -5,8 +5,275 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap&subset=cyrillic,cyrillic-ext,latin-ext" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+
+  <header class="header">
+    <a href="#" class="header__logo"><img src="images/logo.png" alt="Moyo logo"></a>
+    <nav class="menu">
```

Comment:
це не menu, а навігація
```suggestion
    <nav class="nav">
```